src/jti_content.js

```diff
@@ -20,6 +20,10 @@ if (navigator.userAgent.indexOf('Firefox') !== -1 || navigator.userAgent.indexOf
 // Handle <TI> tag selection.
 $(document).on('click', '#description', function () {
     var text = $(this).val(),
+        ctrlDown = false,

```

Comment:
i dont think rate button branch has anything to do with the jti_content.js. Not sure why this file is modified